examples/adaptive_library/DXFImport.py

```diff
@@ -0,0 +1,205 @@
+# imported .dxf geometry will be stored in a list. Use the ``get_region()`` method to get the
+# required regions and store these in the ``standard_regions`` list.
+standard_regions = [
+    mc.get_region("Rotor Pocket_1"),
+    mc.get_region("Rotor Pocket_2"),
+    mc.get_region("Rotor Pocket_4"),
+    mc.get_region("Rotor Pocket_5"),
+    mc.get_region("RotorDuctFluidRegion_2"),
+    mc.get_region("RotorDuctFluidRegion_3"),
+]
+
+# %%
+# Get imported .dxf regions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The imported .dxf regions that we wish to replace the existing regions with will be stored in the
+# a list. Use the ``get_region_dxf()`` method to get the required regions and store in the
+# ``replacement_regions`` list. The indices of the regions in the ``standard_regions`` and
+# ``replacement_regions`` lists must match such that the corresponding standard and replacement
+# regions share the same index (for example, the "RotorDuctFluidRegion_2" and "DXFRegion_Rotor_14"
+# regions are both the fifth element of their respective lists).
+replacement_regions = [
+    mc.get_region_dxf("DXFRegion_Rotor_6"),
+    mc.get_region_dxf("DXFRegion_Rotor_7"),
+    mc.get_region_dxf("DXFRegion_Rotor_10"),
+    mc.get_region_dxf("DXFRegion_Rotor_11"),
+    mc.get_region_dxf("DXFRegion_Rotor_14"),
+    mc.get_region_dxf("DXFRegion_Rotor_12"),
+]
+
+# %%
+# Replace the ``standard_regions`` with the ``replacement_regions``
+# -----------------------------------------------------------------
+# For each standard region, use the ``Region.replace()`` method to replace the region with its
+# corresponding imported .dxf region. The ``Region.replace()`` method replaces the entities
+# (lines and arcs) of a region with entities from another region. The geometry entities will be
+# replaced, while the properties of the original regions are retained (name, material, colour
+# etc.).
+#
+# For more information on the ``Region.replace()`` method see the entry under ``Region`` in
+# :ref:`ref_geometry_functions` in the :ref:`ref_API_Reference`.
+#
+# Once the entities have been replaced, use the ``set_region()`` method to set the modified region
+# in Motor-CAD.
+for index in range(len(standard_regions)):
+    i = standard_regions[index]
+    i.replace(replacement_regions[index])
+    mc.set_region(i)
+
+# %%
+# .. image:: ../../images/DXFImport4.png
+
+# %%
+# Load in Adaptive Templates Script if required
+# ---------------------------------------------
+# When this script is run externally, the following is executed:
+#
+# * Set Geometry type to "Adaptive"
+#
+# * Load the script into the Adaptive Templates tab
+#
+# * Go to the Geometry -> Radial tab to run the Adaptive Templates Script and display the new
+#   geometry
+
+# %%
+# .. note::
+#    When running in a Jupyter Notebook, the path for the Adaptive Templates script (PY file) must
+#    be given instead of ``sys.argv[0]`` when using the ``load_adaptive_script()`` method.
```

Comment:
```suggestion
#    When running in a Jupyter Notebook, you must provide the path for the Adaptive Templates script (PY file)
#    instead of ``sys.argv[0]`` when using the ``load_adaptive_script()`` method.
```
Here script is lowercased. We need to be consistent.